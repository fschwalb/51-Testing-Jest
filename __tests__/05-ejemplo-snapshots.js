
const cliente = {
    nombre: 'Fernando',
    balance: 500,
    tipo: 'Premium'
};



describe( 'Testing al Cliente', () => {

    test( 'Es Fernando', () => {
        expect( cliente ).toMatchSnapshot();
    });

});