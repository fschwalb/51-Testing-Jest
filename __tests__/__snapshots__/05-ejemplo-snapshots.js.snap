// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al Cliente Es Fernando 1`] = `
Object {
  "balance": 500,
  "nombre": "Fernando",
  "tipo": "Premium",
}
`;
